cartoframes/data/observatory/catalog/utils.py

```diff
@@ -65,33 +65,58 @@ def _display_subscription_form_notebook(entity_id, entity_type, info, instant_li
         We will contact you shortly to complete the subscription details.
         '''.format(type=entity_type, days=delivery_days)
 
-    message = '''
-    <h3>Subscription contract</h3>
-    You are about to subscribe to <b>{id}</b>.
-    The cost of this {type} is <b>${price}</b>.
-    If you want to proceed, a Request will be sent to CARTO who will
-    order the data and load it into your account.
-    {delivery_message}
-    In order to proceed we need you to agree to the License of the {type}
-    available at <b><a href="{link}" target="_blank">this link</a></b>.
-    <br>Do you want to proceed?
-    '''.format(
-        id=entity_id,
-        type=entity_type,
-        price=info.get('subscription_list_price'),
-        delivery_message=delivery_message,
-        link=info.get('tos_link'))
-
-    responses = {
-        'ok': '''
-        <b>Congrats!</b><br>The {type} <b>{id}</b> has been requested and it will be available in your account soon.
-        '''.format(id=entity_id, type=entity_type),
-        'cancel': '''
-        The {type} <b>{id}</b> has not been purchased.
-        '''.format(id=entity_id, type=entity_type),
-        'error': '''
-        Subscription error. Please contact to support@carto.com.
-        '''}
+    price = info.get('subscription_list_price')
+    if price is None:
+        message = '''
+        <h3>Data request</h3>
+        You are about to request the {type} <b>{id}</b>.
+        If you want to proceed, a Request will be sent to CARTO who will
+        order the data and load it into your account.
```

Comment:
I would remove these lines as CARTO will not order the data until the customer agrees with the price.

Reply:
OK